### Extracting the Latitude and Longitude of Canada Ontario

In [9]:
import pandas as pd # library for data analsysis
import numpy as np # library to handle data in a vectorized manner

link = "http://cocl.us/Geospatial_data"
df1 = pd.read_csv(link)

df1.head()

Postal Code   Latitude  Longitude
0         M1B  43.806686 -79.194353
1         M1C  43.784535 -79.160497
2         M1E  43.763573 -79.188711
3         M1G  43.770992 -79.216917
4         M1H  43.773136 -79.239476

In [10]:
df1.shape

(103, 3)

the data frames have 103 rows and 3 columns

#### Changing the column name Postal code to Postcode to merge the two data frames together

In [11]:
df1.columns = ['Postcode','Latitude','Longitude']

cols = df1.columns.tolist()
cols

['Postcode', 'Latitude', 'Longitude']

#### Read in the CSV file saved in the previous assignment

In [22]:
link = "https://raw.githubusercontent.com/scorpiocodes/Coursera_Capstone/master/df1.csv"
df = pd.read_csv(link,index_col = 0)
df.head()

Postcode Borough     Neighbourhood
0         Adelaide      M5H  Downtown Toronto
1        Agincourt      M1S       Scarborough
2  Agincourt North      M1V       Scarborough
3   Albion Gardens      M9V         Etobicoke
4        Alderwood      M8W         Etobicoke

#### Merging the two data frames together based on their Postcode

In [0]:
df_new = pd.merge(df, df1, on='Postcode')

#### Save the file as a csv

In [0]:
df_new.to_csv(r'df_final.csv')